### PR TITLE
update location type reference

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?163088
This is why SQL is great for relational data.

### CODE REVIEW

File: corehq/apps/reports/filters/users.py [hunk 1]
```diff
@@ -3,7 +3,8 @@
 from django.utils.translation import ugettext as _
 
 from corehq.apps.es import users as user_es, filters
-from corehq.apps.locations.models import LOCATION_SHARING_PREFIX, LOCATION_REPORTING_PREFIX
+from corehq.apps.locations.models import LOCATION_SHARING_PREFIX, \
```
Comment (dimagimon) [inline corehq/apps/reports/filters/users.py]: F401 'LocationType' imported but unused

**Comment by czue** [comment]
build fail doesn't look related. i'll look into it.

**Comment by czue** [comment]
http://manage.dimagi.com/default.asp?163142

**Comment by czue** [comment]
https://github.com/dimagi/commcare-hq/pull/6088

**Pull request merged**